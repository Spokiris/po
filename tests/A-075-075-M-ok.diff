*** tests/A-075-075-M-ok.out	2023-10-19 17:17:15.000000000 +0100
--- tests/A-075-075-M-ok.outhyp	2023-10-27 13:56:08.379911776 +0100
***************
*** 1,15 ****
- Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: Guardar antes de fechar? Ficheiro sem nome. Guardar ficheiro como: Especifique o número de linhas da folha: Especifique o número de colunas da folha: Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: 
\ No newline at end of file
--- 0 ----
