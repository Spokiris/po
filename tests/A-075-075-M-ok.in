1
s
saved27
34
43
0
