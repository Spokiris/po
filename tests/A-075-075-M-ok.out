Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 1
Guardar antes de fechar? s
Ficheiro sem nome. Guardar ficheiro como: saved27
Especifique o número de linhas da folha: 34
Especifique o número de colunas da folha: 43
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 0
